src/extension/configuration.h

```diff
@@ -109,4 +114,6 @@ DD_CONFIGURATION
 #undef CUSTOM
 #undef CALIAS
 
-#endif  // DD_CONFIGURATION_H
+bool is_config_using_default(dd_config_id id);
```

Comment:
Following the naming conventions, this should be prefixed with `dd_`

Reply:
Done